index.html

```diff
@@ -64,11 +64,11 @@ <h2 class="name">
     </section>
     <section class="work">
```

Comment:
- [ ] I think it would be better if you add on all the navigation items a responsive feature so that when the user click on the navlinks it should redirect them to a corresponding section  🔓 

![Capture 1](https://user-images.githubusercontent.com/80458199/220891574-bd66ecdf-2604-497c-bb7e-b04a0a5080d8.JPG)

Reply:
> * [ ]  I think it would be better if you add on all the navigation items a responsive feature so that when the user click on the navlinks it should redirect them to a corresponding section  🔓
> 
> ![Capture 1](https://user-images.githubusercontent.com/80458199/220891574-bd66ecdf-2604-497c-bb7e-b04a0a5080d8.JPG)

- [ ] Just as the previous reviewer advised, I would follow suit by advising you to add the appropriate ids to where the `#` is so that upon clicking it, it would take the user to the appropriate section.
![image](https://user-images.githubusercontent.com/86200356/220908596-d7499dcc-0488-4c2e-9beb-a3ead300a359.png)